transfermethodui/src/main/res/values/strings.xml

```diff
@@ -28,6 +28,7 @@
     <string name="add_transfer_method_fee_label">Transaction Fee:</string>
     <string name="add_transfer_method_processing_time_label">Processing Time:</string>
 
+    <string name="feeAndProcessingTimeInformation">%s \u2022 %s</string>
```

Comment:
there's fee label after fee value in the mockup `%s fee \u2022 %s`

Reply:
yep thanks 👍